ci-operator/config/openshift/knative-build/openshift-knative-build-release-0.3.yaml

```diff
@@ -1,12 +1,13 @@
 tag_specification:
-  cluster: https://api.ci.openshift.org
-  name: origin-v4.0
-  namespace: openshift
+  name: '4.0'
+  namespace: ocp
+promotion:
+  name: knative-v0.3
+  namespace: ocp
```

Comment:
You need to promote somewhere else.  You aren't part of the OCP release.  Change this back to `openshift`.

Reply:
Fixed. Thanks!